src/member/api.ts

```diff
@@ -112,6 +113,16 @@ export const deleteMember = async (
       .then(({ data }) => data),
   );
 
+export const deleteCurrentMember = async ({
+  API_HOST,
+  axios,
+}: PartialQueryConfigForApi) =>
+  verifyAuthentication(() =>
+    axios
+      .delete<void>(`${API_HOST}/${buildDeleteCurrentMemberRoute()}`)
+      .then(({ data }) => data),
```

Comment:
If void do you need to get the data?

Reply:
It is needed, so the return type is `Promise<void>` and not `Promise<AxiosResponse<void>>`